ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -94,6 +103,7 @@ type State = {
   enableHeaderAnimation: boolean;
   isLongPressEnabled: boolean;
   enableServices: boolean;
+  isWaitingForToast: boolean;
```

Comment:
```suggestion
  isRequestingUserMetadaUpdate: boolean;
```
Could it be more appropriate?

Reply:
state removed within https://github.com/teamdigitale/io-app/commit/98f43b5b9cb3c22ce16fccb1a5065e5cd41fc977. It propose an alternative solution, avoiding:
- the userMetadata upsert being disabled until a previous request has been sent and
- the usage of a component stae to manage the visibility of the toast message.

From my point of view, it could be more aligned to the 'optimistic update' approach.